libs/books/feature/src/lib/reading-list/reading-list.component.ts

```diff
@@ -15,4 +15,8 @@ export class ReadingListComponent {
   removeFromReadingList(item) {
     this.store.dispatch(removeFromReadingList({ item }));
   }
+
+  markAsFinished(item): void {
+    this.store.dispatch(markBookAsFinished({ item, finishedDate:  new Date().toLocaleString() }));
```

Comment:
Should you be using another native Date method here, to make sure you're getting the  ISO date string?

Reply:
Updated accordingly.

<img width="1615" alt="Screen Shot 2020-10-05 at 7 06 52 PM" src="https://user-images.githubusercontent.com/19898265/95144298-f03f1e80-073d-11eb-8791-a92d9c1bb0b8.png">